### PR TITLE
docs: adding an explicit-comms section

### PR DESCRIPTION
**Pull request by madsbk**

### CODE REVIEW

File: docs/source/explicit_comms.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Explicit-comms
+==============
+
+Communication and scheduling overhead can be a major bottleneck in Dask/Distributed. Dask-CUDA addresses this by introducing an API for explicit communication in Dask tasks.
+The idea is that Dask/Distributed spawns workers and distribute data as usually while the user can submit tasks on the workers that communicate explicitly.
```
Comment (pentschev) [inline docs/source/explicit_comms.rst:5]: Maybe we should add what kind of operations/functions can benefit from explicit comms?

Comment (madsbk) [reply docs/source/explicit_comms.rst:5]: Added some more text

**Review by pentschev — APPROVED** [review]
LGTM, thanks @madsbk !

**Comment by madsbk** [comment]
@gpucibot merge

**Pull request merged**